contracts/BSCValidatorSet.sol

```diff
@@ -140,26 +157,30 @@ contract BSCValidatorSet is IBSCValidatorSet, System, IParamSubscriber, IApplica
   event validatorExitMaintenance(address indexed validator);
 
   /*********************** init **************************/
-  function init() external onlyNotInit{
-    (IbcValidatorSetPackage memory validatorSetPkg, bool valid)= decodeValidatorSetSynPackage(INIT_VALIDATORSET_BYTES);
+  function init() external onlyNotInit {
+    (IbcValidatorSetPackage memory validatorSetPkg, bool valid) = decodeValidatorSetSynPackage(INIT_VALIDATORSET_BYTES);
     require(valid, "failed to parse init validatorSet");
-    for (uint i = 0;i<validatorSetPkg.validatorSet.length;i++) {
+    for (uint i = 0; i < validatorSetPkg.validatorSet.length; i++) {
       currentValidatorSet.push(validatorSetPkg.validatorSet[i]);
-      currentValidatorSetMap[validatorSetPkg.validatorSet[i].consensusAddress] = i+1;
+      validatorExtraSet.push(validatorSetPkg._validatorExtraSet[i]);
+      currentValidatorSetMap[validatorSetPkg.validatorSet[i].consensusAddress] = i + 1;
     }
     expireTimeSecondGap = EXPIRE_TIME_SECOND_GAP;
+    finalityRewardRatio = FINALITY_REWARD_RATIO;
     alreadyInit = true;
   }
 
+  receive() external payable {}
+
   /*********************** Cross Chain App Implement **************************/
-  function handleSynPackage(uint8, bytes calldata msgBytes) onlyInit onlyCrossChainContract initValidatorExtraSet external override returns(bytes memory responsePayload) {
+  function handleSynPackage(uint8, bytes calldata msgBytes) onlyInit onlyCrossChainContract external override returns (bytes memory responsePayload) {
```

Comment:
Why delete `initValidatorExtraSet`?

Reply:
Now it will init itself by decode package. No need modifier